src/ui/backend/BaseBacking.js

```diff
@@ -62,36 +83,39 @@ export default class BaseBacking {
 
   copy () {
     var copy = {};
+    for (var key in BASE_STYLE_PROPS) {
```

Comment:
They recently improved the performance of ```for ... in``` loops in v8 (https://v8project.blogspot.jp/2017/03/fast-for-in-in-v8.html), not sure about other engines.
I was benchmarking it a few month ago in chrome and noticed that yes, it is pretty similar to regular loops. The thing is, older browser version might be slow.
I can use an array instead.